### DISCUSSION TITLE
Why all caps CONTRIBUTING and LICENSE?

### DISCUSSION THREAD

**Discussion by LadyAleena** [community/community#22673] [New to GitHub]
<p>Why do the files CONTRIBUTING and LICENSE have to be in all caps to trigger certain aspects of GutHub? While I know there probably needs to only be one of each, I don’t like my file names yelling at me. Plus, my caps lock key is assigned to something else on my system, so typing anything in all caps is a pain in my pinkies.</p>
<p>So, please allow lowercase contributing and license file names to trigger things?</p>

**Comment by ernest-phillips** [comment, marked as answer]
<p>Thanks for expressing your concerns <a class="mention" href="/ladyaleena">@LadyAleena</a> . I can see how the default casing can be an issue in that regard.<br>
Those files default to all-uppercase to help promote visibility within your repository. Having all-caps makes it visually easier to see, and denote prominence.</p>
<p>You are also not obligated to use all-caps for these files. They will be counted for your Community file with lower-case as well.</p>

**Comment by woutervh** [comment]
IF UPPERCASING is not MANDATORY, please state so in the docs, instead of leaving it to trial& error to the end-user.

**Reply by rphair** [reply]
Would you please INDICATE where IN THE DOCS this omission has been made, e.g. with a LINK? (it would be appreciated by our own project: there are some other aspects of this behaviour we are trying to reverse-engineer) 🙏

**p.s.** I found it, after long & agonising search: https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file

**Comment by ooker777** [comment]
Simply copying the answer in this question in Law Stack Exchange: [In Contracts, why is some text all in uppercase?](https://law.stackexchange.com/a/18210/26060)

> It is actually because "this is important". Under US law, disclaimers must be "conspicuous" ([UCC 2-316][1]). So you can talk regularly when you're just stating the terms, but if you're disclaiming liability, YOU MUST BE CONSPICUOUS ("to exclude or modify the implied warranty of merchantability or any part of it the language must mention merchantability and in case of a writing must be conspicuous, and to exclude or modify any implied warranty of fitness the exclusion must be by a writing and conspicuous"). There are many ways to make text conspicuous, so bold or larger type would do, but all-caps is pretty bullet-proof from a technological perspective.
>
> Thanks to ohwilleke for salient citations: invalidation of a [plain-type buried indemnification clause][2], [all-caps clause][3] held to be sufficient, [law review article][4] on the conspicuousness requirement.


  [1]: https://www.law.cornell.edu/ucc/2/2-316
  [2]: http://ca10.uscourts.gov/opinions/13/13-3076.pdf
  [3]: http://cases.justia.com/federal/appellate-courts/ca10/06-6290/06-6290-2011-03-14.pdf?ts=1411084674
  [4]: https://mjlr.org/2015/09/19/out-of-sight-out-of-mind-hidden-disclaimers-and-ucc-%C2%A7-2-316s-conspicuousness-requirement/
